ui/src/main/scripts/plugins/directives/instance-view-diagram/ts/instanceViewDiagramDirective.ts

```diff
@@ -198,7 +198,7 @@ module InstanceViewDiagram {
           _.each(d.properties, (property) => {
             if (!!property.value) {
               nodeTT += ('<li class=\'tt-prop\'><strong>' + property.name + '</strong> ' + property.value + '</li>');
-              if (property.name === 'fault') {
+              if (property.name === 'fault' || property.name === 'error') {
```

Comment:
Or be even more bulletproof by `prop.name === 'error && prop.val !== false'`? Tag value should be boolean, but what one accidentally used it with a string value?